### PR TITLE
String optimizations for English language

### PR DESCRIPTION
**Pull request by atrol**
Fixes [#28905](https://mantisbt.org/bugs/view.php?id=28905)

### CODE REVIEW

**Review by dregad — CHANGES_REQUESTED** [review]
While I'm fine with removing the "Manage" prefix from the tabs and most of the links as it is redundant

image

bear in mind that these strings are also used as page title so the browser tab would now e.g. display `Users` instead of `Manage Users` which IMO was better.

image

File: lang/strings_english.txt [hunk 1]
```diff
@@ -667,7 +667,7 @@ $s_rss = 'RSS';
 # manage_overview_page
 $s_site_information = 'Site Information';
 $s_mantis_version = 'MantisBT Version';
-$s_schema_version = 'Schema Version';
+$s_schema_version = 'Database Schema Version';
```
[the remark this answers is not in the document]
Comment (atrol) [reply lang/strings_english.txt:670]: Obvious for you and me, but not for MantisBT newbies.
Furthermore we use `Database Schema` at various other places.

**Comment by atrol** [comment]
> While I'm fine with removing the "Manage" prefix from the tabs and most of the links as it is redundant
> bear in mind that these strings are also used as page title so the browser tab would now e.g. display `Users` instead of `Manage Users` which IMO was better.

@dregad right, the title was better.
I am not sure if introducing separate strings for link and title and rewriting some functions is a good idea.
As I don't have time to implement something like that at the moment, there are two options.
- nice labels on the tabs
- nice page titles

Let me know what you prefer

File: lang/strings_english.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (dregad) [reply lang/strings_english.txt:670]: OK

**Comment by dregad** [comment]
> I am not sure if introducing separate strings for link and title and rewriting some functions is a good idea.

I agree, it's not worth the effort for sure, and probably not such a good idea.

> Let me know what you prefer

A bit of a tough call... 
- Personally, I like having descriptive page titles, so the current display which makes it very clear what the tab is about suits me fine. 
- On the other hand, it is also somewhat obvious/implied if I read e.g. *User - MantisBT* or *Plugins - MantisBT* that I'm on the management page, as there are no other places.

So maybe I just need to get used to the change... Let me think about it some more and I'll get back to you (and do remind me if I don't me respond in the next couple weeks :wink: )

**Comment by vboctor** [comment]
I'm in favor of this change.

**Comment by atrol** [comment]
@dregad last chance to object, will merge tomorrow.

**Comment by dregad** [comment]
>
> @dregad <https://github.com/dregad> last chance to object, will merge
> tomorrow.
>
Sorry, I said in a couple weeks (meaning 2) as I'm currently on vacation
with other things to think about than Mantis 😉

I'll look at it when I'm back home (Monday).

>

**Comment by atrol** [comment]
No rush, enjoy your vacation.

**Review by dregad — APPROVED** [review]

**Pull request merged**